pod-resizer/README.md

```diff
@@ -0,0 +1,29 @@
+# pod-resizer
+
+This container image watches over another container in a deployment, and
+vertically scales the dependent container up and down. Currently the only
+option is to scale it linearly based on the number of nodes, and it only works
+for a singleton.
+
+## Nanny program and arguments
+
+The nanny scales resources linearly with the number of nodes in the cluster. The base and marginal resource requirements are given as command line arguments, but you cannot give a marginal requirement without a base requirement.
```

Comment:
Like an example yaml file? I have one just lying around I could add if that's what you want.